_data/simple-icons.json

```diff
@@ -125,6 +125,11 @@
             "hex": "000000",
             "source": "https://www.apple.com/itunes/marketing-on-music/identity-guidelines.html#apple-music-icon"
         },
+        {
+            "title": "Apple Pay",
+            "hex": "000000",
```

Comment:
From the [landing page](https://www.apple.com/apple-pay/) and the logo found their, I would say this should be `#111111`, the SVG source you linked actually doesn't specify a color at all, and so it defaults to `#000000`

Reply:
The assets available on Apple's site: https://developer.apple.com/apple-pay/marketing/ use #000000.